<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPINHYDW0834" ignored="0" name="Default test" tests="9" failures="1" timestamp="17 Jan 2018 10:18:37 GMT" time="28.507" errors="0">
  <testcase name="addImplicitly" time="1.159" classname="com.epam.tam.module.tests.RunnerTest"/>
  <testcase name="verifyHomePage" time="7.271" classname="com.epam.tam.module.tests.RunnerTest">
    <failure type="junit.framework.ComparisonFailure" message="Home page not opened Successfully expected:&amp;lt;...z...&amp;gt; but was:&amp;lt;......&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: Home page not opened Successfully expected:<...z...> but was:<......>at junit.framework.Assert.assertEquals(Assert.java:81)at com.epam.tam.module.tests.RunnerTest.verifyHomePage(RunnerTest.java:56)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- verifyHomePage -->
  <testcase name="verifyOriginValue" time="0.57" classname="com.epam.tam.module.tests.RunnerTest"/>
  <testcase name="verifyDestinationValueNegative" time="0.2" classname="com.epam.tam.module.tests.RunnerTest"/>
  <testcase name="verifyDestinationValue" time="0.354" classname="com.epam.tam.module.tests.RunnerTest"/>
  <testcase name="selectDepartureDate" time="0.42" classname="com.epam.tam.module.tests.RunnerTest"/>
  <testcase name="selectReturnDate" time="3.238" classname="com.epam.tam.module.tests.RunnerTest"/>
  <testcase name="verifyPassengers" time="2.337" classname="com.epam.tam.module.tests.RunnerTest"/>
  <testcase name="verifySearchPage" time="7.732" classname="com.epam.tam.module.tests.RunnerTest"/>
</testsuite> <!-- Default test -->
